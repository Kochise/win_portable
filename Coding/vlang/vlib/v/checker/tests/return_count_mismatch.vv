fn v() {
	return 3
}

fn f() int
{
	return
}

fn g() (int, string)
{
	return
}

fn ff() int
{
	return 2, ''
}

fn gg() (int, string)
{
	return 3
}

