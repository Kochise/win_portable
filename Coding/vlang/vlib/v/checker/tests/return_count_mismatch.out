vlib/v/checker/tests/return_count_mismatch.vv:2:9: error: unexpected argument, current function does not return anything
    1 | fn v() {
    2 |     return 3
      |            ^
    3 | }
    4 |
vlib/v/checker/tests/return_count_mismatch.vv:7:2: error: expected `int` argument
    5 | fn f() int
    6 | {
    7 |     return
      |     ~~~~~~
    8 | }
    9 |
vlib/v/checker/tests/return_count_mismatch.vv:12:2: error: expected `(int, string)` arguments
   10 | fn g() (int, string)
   11 | {
   12 |     return
      |     ~~~~~~
   13 | }
   14 |
vlib/v/checker/tests/return_count_mismatch.vv:17:2: error: expected 1 argument, but got 2
   15 | fn ff() int
   16 | {
   17 |     return 2, ''
      |     ~~~~~~~~~~~~
   18 | }
   19 |
vlib/v/checker/tests/return_count_mismatch.vv:22:2: error: expected 2 arguments, but got 1
   20 | fn gg() (int, string)
   21 | {
   22 |     return 3
      |     ~~~~~~~~
   23 | }
   24 |
